### ISSUE TITLE
Commands with modifiers don't work with violations with line-only locations

### ISSUE THREAD

**Issue by jpsim** [realm/SwiftLint#316]
Easier explained with an example. This shouldn't trigger a violation, but it does:

```swift
// swiftlint:disable:next line_length
/* a very long line. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi est enim, vestibulum a arcu in, congue volutpat sapien. Proin nec mi tellus. Vestibulum blandit risus a lobortis dignissim. Ut vulputate placerat felis, in malesuada risus accumsan sit amet. Suspendisse non purus sit amet odio porttitor egestas. Vestibulum eu metus elit. Curabitur cursus ante quis ornare consequat. Nam faucibus at nunc a commodo. Donec ultricies facilisis ante. Curabitur placerat metus ut elit sodales eleifend. Aliquam blandit nunc in mauris lobortis vulputate. Proin venenatis congue neque sed ullamcorper. Mauris ac aliquam mauris. Mauris feugiat dignissim consequat. Etiam mattis neque mattis, aliquet risus vel, condimentum erat.*/
```

This happens because the style violation's location is set to `line: 2, character: nil` although the commands in the file above are set to disable `line_length` as of `line: 2, character: 0` (e.g. after the violation location).
